ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -297,6 +297,21 @@ def execution_environment_volume_mounts(
             entry.value.current = parsed_volume_mounts
         return messages, exit_messages
 
+    @staticmethod
+    @_post_processor
+    def container_options(
+        entry: Entry, config: ApplicationConfiguration
+    ) -> PostProcessorReturn:
+        # pylint: disable=unused-argument
```

Comment:
This looks out-of-place. Normally, you should place ignores on lines violating them.

Reply:
> This looks out-of-place. Normally, you should place ignores on lines violating them.

It conforms to the placement elsewhere in this file - all of the other methods that use the ignore have the ignore comment as the first line in the method - unless you want me to refactor the entire file to ensure that all of the pylint ignores used in the file are only placed on the lines where they are used

Reply:
Ah, never mind. It's a wider problem then.